packages/cohort/src/replicator/pg_replicator_installer.rs

```diff
@@ -0,0 +1,28 @@
+// $coverage:ignore-start
+use std::io::Error;
+
+use async_trait::async_trait;
+use log::info;
+
+use crate::{state::postgres::data_access::PostgresApi, tx_batch_executor::BatchExecutor};
+
+use super::core::{ReplicatorInstaller, StatemapItem};
+
+pub struct PgReplicatorStatemapInstaller {
+    pub pg: PostgresApi,
+}
+
+#[async_trait]
+impl ReplicatorInstaller for PgReplicatorStatemapInstaller {
+    async fn install(&mut self, sm: Vec<StatemapItem>, version: Option<u64>) -> Result<bool, Error> {
+        info!("Last version ... {:#?} ", version);
+        info!("Original statemaps received ... {:#?} ", sm);
```

Comment:
these two should be debug. When we run high load and measure perf, that info will show up a lot